schemas/json/block.json

```diff
@@ -73,6 +73,13 @@
 				"type": "string"
 			}
 		},
+		"children": {
```

Comment:
Might be nit-picking but I feel like a more descriptive name would be better. E.g. `validChildren` or `allowedChildren` would indicate its a property imposing limitations/constraints on the block.

I was just thinking about people using `.children`, I can easily see people thinking "_Are these my blocks actual children, or allowed children?_".

I don't feel super strongly about this so if you think it's fine as-is I'm open to that.

Reply:
I wanted to be consistent with the existing `parent` and `ancestor` fields. They aren't named `validParents` or `allowedAncestors` either. That's why I chose the plain `children`.

Reply:
Yeah, makes sense. Like I said, was a nit-pick above anything else so fine with me. 👍🏻